moulinette/interfaces/api.py

```diff
@@ -80,6 +80,24 @@ def set_defaults(self, **kwargs):
     def get_default(self, dest):
         return self._parser.get_default(dest)
 
+    def add_arguments(self, arguments, extraparser, format_arg_names=None, validate_extra=True):
```

Comment:
Ah, yes it is, because the _HTTPArgumentParser is what's returned by add_action_parser. Hm... Wondering if that's worth factorizing this in a mother class like 'MoulinetteActionParser' to avoid copy-pasta of the add_arguments... But maybe that's good enough.